package.json

```diff
@@ -1,11 +1,19 @@
 {
-  "name": "s47_nkr_plu2_0",
+  "name": "nodejs-with-express",
   "version": "1.0.0",
-  "description": "PROJECT TITLE",
-  "main": "index.js",
+  "description": "",
```

Comment:
Add a meaningful description to your package.json file. This helps others understand what your project is about. [medium]